libs/general/src/lib/user-not-allowed-access/user-not-allowed-access.component.html

```diff
@@ -0,0 +1,16 @@
+<h1 mat-dialog-title class="mt-2 mb-2">
+  {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.TITLE' | translate}}
+</h1>
+
+<h2 mat-dialog-content class="mt-2 mb-2">
+  {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.MESSAGE' | translate}}
+</h2>
+
+<mat-dialog-actions align="end">
+  <button mat-flat-button color="primary" (click)="redirectToLogin()">
+    {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.LOGIN' | translate}}
+  </button>
+  <button mat-flat-button color="primary" (click)="redirectToServiceLogin()">
+    {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.SERVICE_LOGIN' | translate}}
+  </button>
+</mat-dialog-actions>
```

Comment:
Fixes the dialog, layout and colors.
```suggestion
<h1 mat-dialog-title>{{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.TITLE' | translate}}</h1>
<div mat-dialog-content class="dialog-container">
  {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.MESSAGE' | translate}}
</div>
<div mat-dialog-actions>
  <button mat-flat-button class="ms-auto" color="accent" (click)="redirectToLogin()">
    {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.LOGIN' | translate}}
  </button>
  <button mat-flat-button color="accent" (click)="redirectToServiceLogin()">
    {{'SHARED_LIB.PERUN.COMPONENTS.USER_NOT_ALLOWED.SERVICE_LOGIN' | translate}}
  </button>
</div>
```